tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -167,8 +175,12 @@ stages:
 
         steps:
         # Setup
-        - checkout: ff_pipeline_host
-          clean: true
+        - ${{ if eq(parameters.checkout, true) }}:
+          - checkout: self
+            clean: true
+        - ${{ else }}:
+          - checkout: ff_pipeline_host
+            clean: true
```

Comment:
I suspect that now we need to checkout `ff_pipeline_host` no matter what, and the param would do (or not) a checkout of `self`. I don't recall why exactly it was necessary to check out the public repo... access to the Dockerfile to start the local environment, probably?